src/njoy21/input/VIEWR/Card1/Infile.hpp

```diff
@@ -0,0 +1,24 @@
+struct Infile {
+  using Value_t = int;
+
+  static std::string name() { return "infile"; }
+
+  static std::string description() {
+	return 
```

Comment:
Please indent 2 space, not 4. See https://njoy.github.io/DevelopersGuide/Style/Code.html